packages/contractkit/src/identity/odis/query.ts

```diff
@@ -136,6 +138,10 @@ export async function queryOdis<ResponseType>(
         case 401:
           throw new Error(ErrorMessages.ODIS_AUTH_ERROR)
         default:
+          if (Math.floor(res.status / 100) === 4) {
```

Comment:
Any reason we can't do res.status >= 400 && <= 500 like Cody suggested?

Reply:
Oh I didn't see that suggestion. That works for me